### DISCUSSION TITLE
Sistema Antifurto con finestre aperte

### DISCUSSION THREAD

**Discussion by gup133** [indomus/forum#2941] [GENERALE - Discussioni generiche]
Ciao a tutti, vorrei realizzare un sistema di antifurto fai da te (rigorosamente wifi) che mi permetta, le notti d'estate, di lasciare aperte le finestre (e le persiane in ferro) per far passare aria. Sostanzialmente, quello di cui avrei bisogno è:
Base/Sirena Interna
Sirena Esterna
Sensori di Movimento
Sensori per finestre (che funzionino anche in caso di finestre lasciate aperte)

Il problema, appunto, è trovare dei sensori wifi che mi permettano, la notte, di lasciare aperte le persiane pur in presenza dell'antifurto allarmato. Avevo pensato, per ovviare al problema, ai sensori antivibrazione somfy, ma perchè funzionino dovrei tenere le persiane non completamente aperte, al max accostate e unite con una staffa in modo che in caso di effrazione, per entrare si dovrebbero comunque spostare le persiane permettendo all'antifurto di suonare.
Avete idee e/o consigli?

**Comment by indomus** [comment]
ma usi un HUB personale o, di preciso, cosa cerchi come centralizzazione?

**Comment by gup133** [comment]
Al momento non ho messo in piedi alcun hub, nella mia abitazione principale ho centralizzato riscaldamento/condizionatori/allarme/tapparelle/luci attraverso HA, però é la casa di villeggiatura estiva e per ora non ho pianificato nulla, anche se in futuro credo che provvederò ad implementare anche lì qualcosa del genere. Il problema é che non posso fare opere murarie (per cui devo pensare a qualcosa di wifi) e che non ho inferriate, per cui devo trovare qualcosa che possa "proteggermi" anche con persiane aperte

**Reply by indomus** [reply]
valuta l'allarme di Amazon Ring. sensori di apertura che funzionino anche con le finestre aperte naturalmente non esistono

**Reply by gup133** [reply]
Grazie mille per la risposta. Il ring l'ho montato a casa dei miei, onestamente, per essere un piccolo antifurto All in one, mi è piaciuto: rapidità e facilità di installazione, configurazione semplicissima anche per i miei che con le app non hanno molta dimestichezza (il tastierino basico li aiuta moltissimo). Il problema è che i sensori sono quelli classici a contatto. Nel mio caso, se volessi lasciare aperte le persiane per avere più aria durante la notte non potrei allarmare casa in quanto il sensore riconoscerebbe un tentativo di apertura (come realmente è). Per questo avevo pensato a somfy, hanno gli intellitag che essendo sensori anti vibrazione, possono essere allarmati anche a persiana aperta (fanno partire l'allarme in caso di movimento/vibrazione della persiana a prescinedere dallo stato aperto/chiuso della stessa). Solo che volevo sapere se esiste qualcosa del genere anche di altre case, in quanto somfy, benchè integrabile in HA mediante MQTT non è integrabile in alexa

**Reply by indomus** [reply]
con ring alarm puoi serenamente indicargli di forzare l'attivazione dell'allarme con determinati sensori a contatto in stato di apertura, perché dici di no?

**Comment by gup133** [comment]
> con ring alarm puoi serenamente indicargli di forzare l'attivazione dell'allarme con determinati sensori a contatto in stato di apertura, perché dici di no?

Davvero? dall'app non mi sono accorto di questa opzione. Da dove posso farlo?
Ma nel caso, potrei dirgli di forzare lo stato di apertura per esempio in camera da letto e nel caso qualcuno provasse a muovere le persiane suonerebbe lo stesso? Perchè ciò che vorrei è poter dormire con finestre/persiane aperte avendo comunque un minimo di protezione (per esempio accostando le persiane pur lasciandole aperte in maaniera che se qualcuno provasse ad entrare dovrebbe comunque spostare un pò le persiane facendo partire i sensori....per questo cercavo qualcosa tipo intellitag di somfy, perchè in questo caso l'unico modo per essere avvertito dalla sirena di allarme è che qualcuno muova le persiane a prescindere dal loro stato di aperto/chiuso)

**Reply by indomus** [reply]
guarda io non l'abbiamo sottomano ma siamo sicuri si possa fare. se lasci le finestre aperte, l'unica soluzione per stare sereni è mettere di fronte alla finestra un sensore di presenza, ma è chiaro che non ci deve passare davanti nessuno, altrimenti scatta.

**Reply by gup133** [reply]
Grazie ancora per la pazienza.
Riporto dal sito della Ring (non posto il link per evitare di infrangere il regolamento):
_IGNORARE I SENSORI: se un sensore di contatto è aperto oppure offline al momento dell'inserimento dell'allarme o se desideri lasciare aperta una finestra, è possibile bypassare il sensore di contatto per lasciarlo disattivato._
In questo caso si "disabilita" (ignora) il sensore, mantenendo il sistema di allarme comunque attivato su tutti gli altri componenti. Però così, lasciando la finestra aperta con il relativo sensore ignorato, eventuali tentativi di effrazione non verrebbero rilevati.
come giustamente dici tu, l'unico modo per proteggersi con finestre aperte sarebbe quello di mettere un sensore di movimento vicino la finestra, ma temo sarebbe poco applicabile perchè essendo vicini a letto i falsi allarmi temo si sprecherebbero.
Per questo cercavo qualcosa di diverso, o un sensore vibrazione (tipo il somfy) oppure qualcosa che, passando attraverso la finestra, suoni. Ho letto di sensori PIR a tecnologia mista...ma onestamente non ho trovato nulla che faccia al caso mio (nel senso di un sistema di allarme "all in one" che abbia sensori di movimento, sensori apertura/chiusura finestre, sirene, sensori PIR a tecnologia mista etc etc)

**Reply by indomus** [reply]
se la finestra è aperta è normale che quel sensore debba venire ignorato, e appunto ring ti permette di farlo. eventuali tentativi di effrazione non esistono proprio concettualmente, dato che la finestra è già aperta.

**Comment by gup133** [comment]
> se la finestra è aperta è normale che quel sensore debba venire ignorato, e appunto ring ti permette di farlo. eventuali tentativi di effrazione non esistono proprio concettualmente, dato che la finestra è già aperta.

Si infatti, ho sbagliato termine...concettualmente servirebbe un controllo dei varchi (finestre) in modo che l'allarme si attivi quando qualcuno varca la soglia della finestra...

**Reply by indomus** [reply]
capito. hm, difficile trovarne su queste categorie di prezzo.

**Comment by gup133** [comment]
> capito. hm, difficile trovarne su queste categorie di prezzo.



> capito. hm, difficile trovarne su queste categorie di prezzo.
Nel caso uno volesse anche salire di livello (di prezzo), di cosa staremmo parlando? In materia conosco poco...

**Reply by indomus** [reply]
qui si esce dal tema squisitamente domotico a quello di allarmistica professionale, sulla quale non siamo ferrati ed è cmq OT rispetto ai temi di cui sopra... :)
